holmes/plugins/toolsets/aks-node-health.yaml

```diff
@@ -20,23 +20,23 @@ toolsets:
         command: |
           kubectl describe node {{ NODE_NAME }}
 
-      - name: "check_kubelet_logs"
-        description: "Fetches kubelet logs from a specific node in the AKS cluster."
-        user_description: "get kubelet logs for node {{ NODE_NAME }}"
+      - name: "get_node_events"
+        description: "Fetches recent events for a specific node to surface warnings and errors."
+        user_description: "get events for node {{ NODE_NAME }}"
         command: |
-          ssh {{ NODE_NAME }} 'journalctl -u kubelet'
+          kubectl get events --field-selector involvedObject.kind=Node,involvedObject.name={{ NODE_NAME }} --sort-by='.lastTimestamp'
```

Comment:
I think it's more like a runbook instead of tools in toolsets. 
A better practice for the aks node health case I think is we introduce a prompt/runbook like the dns trouble guide we have, and leverage the existing tools if we have, or introduce new tools to get resources.

https://github.com/robusta-dev/holmesgpt/blob/5cc6a34c18ac548e0bfa5d6a80213bbb5baebc4c/holmes/plugins/toolsets/kubernetes.yaml#L41-L43

Reply:
what do you think @arikalon1